cellpack/bin/simularium_converter.py

```diff
@@ -3,12 +3,19 @@
 import traceback
 
 # import cellpack.autopack.transformation as tr
-from collections import OrderedDict
 import numpy as np
 import json
 import logging
+from scipy.spatial.transform import Rotation as R
```

Comment:
this is the only file that has actual code changes, everything else is formatting fixes